test/unit_tests/braket/circuits/test_circuit.py

```diff
@@ -811,6 +811,48 @@ def test_from_ir_with_measure():
     assert Circuit.from_ir(source=ir.source, inputs=ir.inputs) == expected_circ
 
 
+def test_from_ir_with_single_measure():
+    ir = OpenQasmProgram(
+        source="\n".join(
+            [
+                "OPENQASM 3.0;",
+                "bit[2] b;",
+                "qubit[2] q;",
+                "h q[0];",
+                "cnot q[0], q[1];",
+                "b = measure q;",
+            ]
+        ),
+        inputs={},
+    )
+    expected_circ = Circuit().h(0).cnot(0, 1).measure(0).measure(1)
+    assert Circuit.from_ir(source=ir.source, inputs=ir.inputs) == expected_circ
+
+
+def test_from_ir_round_trip_transformation():
+    circuit = Circuit().h(0).cnot(0, 1).measure(0).measure(1)
+    ir = OpenQasmProgram(
+        source="\n".join(
+            [
+                "OPENQASM 3.0;",
+                "bit[2] b;",
+                "qubit[2] q;",
+                "h q[0];",
+                "cnot q[0], q[1];",
+                "b[0] = measure q[0];",
+                "b[1] = measure q[1];",
+            ]
+        ),
+        inputs={},
+    )
+    new_ir = circuit.to_ir("OPENQASM")
+    new_circuit = Circuit.from_ir(new_ir)
+
+    assert new_ir == ir
+    assert Circuit.from_ir(source=ir.source, inputs=ir.inputs) == circuit
```

Comment:
This is already covered by other test cases, right? Not that I'm complaining.

Reply:
Yes, it is covered by other test cases